src/reducers/table-utils/kepler-table.ts

```diff
@@ -19,50 +19,36 @@
 // THE SOFTWARE.
 
 import {console as Console} from 'global/console';
-import {TRIP_POINT_FIELDS, SORT_ORDER, ALL_FIELD_TYPES, SCALE_TYPES} from '@kepler.gl/constants';
 import {ascending, descending} from 'd3-array';
 
 // import {validateInputData} from 'processors/data-processor';
-import {generateHashId} from 'utils/utils';
-import {getGpuFilterProps, getDatasetFieldIndexForFilter} from 'utils/gpu-filter-utils';
+import {TRIP_POINT_FIELDS, SORT_ORDER, ALL_FIELD_TYPES, SCALE_TYPES} from '@kepler.gl/constants';
+import {RGBColor, Field, FieldPair, FieldDomain, Filter, ProtoDataset} from '@kepler.gl/types';
+
+import {generateHashId, getSortingFunction} from '@kepler.gl/utils';
+import {getGpuFilterProps, getDatasetFieldIndexForFilter} from '../layer-utils/gpu-filter-utils';
```

Comment:
`getGpuFilterProps, getDatasetFieldIndexForFilter` should be moved to `table-utils`